dynatrace_topology/stackstate_checks/dynatrace_topology/dynatrace_topology.py

```diff
@@ -301,4 +299,4 @@ def get_dynatrace_json_response(self, endpoint, timeout=10):
                 return yaml.safe_load(resp.text)
```

Comment:
@TizianoPerrucci do we want to swap this as well with `json.load` as in tests you already converted to `json.load`?

Reply:
@vishalxebia yes next ticket, another PR coming

Reply:
ok.